# DEF-D Fysieke Ontwerpopdracht 2 invul template

## Groep informatie

Aan het einde van deze ontwerpopdracht lever je als groep één gezamenlijk notebook in (als in *PR* op GitHub en de link op Brightspace). Zet jullie groepsnummer, studienummers en namen in onderstaande tabel zodat wij zeker weten dat het groepsvormen goed is gegaan.

| Groep: 40   |  |
| :-------------|:-------------|
| Xin Qi Huang | 6345735|
| Aren Meghdadi| 6474349 |
| Lars Mulder  | 6546862 |
| Planning Groep: 40    | |
|---|---|
| Mijlpaal 1:  sanity check 1 en 2                | 12:00 |
| Mijlpaal 2:  sanity check 3                     | 12:30 |
| Mijlpaal 3:  synthese en evalutaie              | 14:30 |
| MIjlpaal 4:  simulatie en begin fabricge        |15:30|
| Mijlpaal 5:  sensor gemaakt en metingen gedaan  | 16:30 |
| Pauze 1| 12:30-13:30 |
| Pauze 2| 15:45-16:00 |

## *Opdracht 1*: Sanity check 3

#### Voeg hier een foto van de schets van je plaatcondensator in. Geef daarbij ook duidelijk de waarde van $A$ en $d$ aan.
<img src="tudelftLogo.png" alt="schets student 1" width="400"/>

In [ ]:
# Schrijf hier een python functie die de capaciteit van een plaatcondensator uitrekent. Zorg dat de functie
# als input de ontwerpparameters van de vlakke plaat formule accepteert (oppervlakte, afstand en diëlektrische
# constante) en als output de capaciteit van de vlakke plaat condensator met die ontwerpparameters.
# Je gaat deze formule de rest van de dag gebruiken: controleer hem goed!
from scipy.optimize import curve_fit
import numpy as np
import matplotlib.pyplot as plt 

# constante
epsilon_0 = 8.85e-12
epsilon_r = 1

# capaciteit berekenen 
def vlakkeplaat(A, d, epsilon_r):
    return epsilon_0 * epsilon_r * A / d

# dikte om 50 pF te bereiken 
d = (epsilon_0 * epsilon_r * 0.01) / 50e-12
print(f'De dikte voor 50 pF moet zijn {d:.4f} m') 


De dikte voor 50 pF moet zijn 0.0018 m


In [44]:
# Gebruik je formule om uit te rekenen wat je verwacht dat de capaciteit van jullie vlakke plaat condensatoren
# gaat zijn.
C = vlakkeplaat(0.01,1e-3,epsilon_r=3)
C_pf = C*1e12

print(f"Berekende verwachte capaciteit plaatcondensator groep 40 is {C_pf:.1f} pF")



Berekende verwachte capaciteit plaatcondensator groep 40 is 265.5 pF


#### Voeg hier een foto van je plaatcondensator en van de meting op de scope in. Dus 2 foto's.
![scope meting 1](condensator.jpeg)
![scope meting 2](oscilloscoop.jpeg)

### Meting capaciteit
Vul onderstaande tabel in. Bij Sanity check 2 heb je afgeleid hoe je van berekende capaciteit naar berekend voltage gaat en terug. 


| Namen         | Berekende capaciteit   | Berekend Voltage | Gemeten Voltage | Gemeten capaciteit   |
|-------------|-------------:|-------------:|-------------:|-------------:|
| groep 40| 265.5 pF |1.33 V | 0.380 V |75.6 pF |

De gemeten capaciteit is kleiner doordat we de parameters niet goed geschat hebben en de draden ook invloed hebben op de capaciteit(waar de berekening geen rekening mee houd). 

## *Opdracht 2:* Synthese.

#### Voeg hier voor een foto van je schets(en) van minimaal drie sensoren in.
![drie schetsen](tudelftLogo.png)


## Opdracht 3: Welke sensor gaan jullie maken?

Pas deze tekst aan: Wij gaan een sensor maken die X meet.

## Opdracht 4: Uitgewerkte schets (afmetingen!) en plot van de ontwerpgrafiek.

#### Voeg hier een foto van jullie uitgewerkte  schets van de sensor die jullie gaan maken in
![uitgewerkte schets](tudelftLogo.png)

In [3]:
# Plot hier de verwachte ontwerpgrafiek van jullie sensoren.

## Opdracht 5: Invullen tabel LTSpice.

|         | Capaciteit waarde (pF)  | LTSpice berekend output Vpp (mV)|
|-------------|-------------:|-------------:|
| minmale capaciteit| ? |? |
| gemiddelde capaciteit| ? |? |
| maximale capaciteit| ? |? |

## Opdracht 6: Foto's werkende opstelling.

#### Voeg hier de foto's van je sensoren in actie in.
![meting 1](tudelftLogo.png)
![meting 2](tudelftLogo.png)

## Opdracht 7: Plot de kalibratiegrafiek.

In [4]:
# Plot hier je gemeten kalibratiegrafiek en plot in dezelfde figuur ook je verwachte ontwerpgrafiek.

## Opdracht 8: Vul de tekst aan.

Onze sensor kan nog verbetered worden door de volgende verbeteringen (minimaal 2):

* verbetering 1
* verbetering 2

## Opdracht 9: Uiteindelijke ontwerp en kalibratiegrafiek.

#### Voeg hier een foto van je uiteindelijke sensor in.
![foto sensor](tudelftLogo.png)

In [5]:
# Plot hier je oorspronkelijke en je uiteindelijke kalibratiegrafieken, fit daar een bijpassende lijn door
# en zet ook de verwachte ontwerpgrafiek erbij

## Opdracht 10: Vul hieronder de conclusie in.

Vul hier de conclusie in

## Opdracht 11: Kopieer hieronder de leerdoelen.
1. Het kunnen gebruiken van de eigenschappen van een elektrische component in een ontwerpproces om een fysische grootheid te meten.
2. Kennis hebben van gedrag van elektrische componenten en meetopstellingen.
3. Onderdelen van een datasheet zoals een kalibratiegrafiek opstellen, gevoeligheid en onzekerheid beschrijven.

kopieer hier de cel(len) waaruit blijkt dat jullie leerdoel 1 beheersen.

kopieer hier de cel(len) waaruit blijkt dat jullie leerdoel 2 beheersen.

kopieer hier de cel(len) waaruit blijkt dat jullie leerdoel 3 beheersen.